invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/InvitationActionButtons.js

```diff
@@ -35,9 +34,14 @@ export const ActionButtons = ({
           onClick={onReInvite}
         />
       )}
-      {showViewButton && (
-        <Button size="tiny" content={i18next.t("View")} onClick={onView} />
-      )}
+
+      <Button
```

Comment:
I would suggest reverting these changes, since the Invitation search results will be refactored